### PR TITLE
Rails on vue/chapter three

### PR DESCRIPTION
**Pull request by ctrlaltdelete44**

### CODE REVIEW

File: collections/_blog-posts/04-rails-with-vue-updating-rails-data.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
+---
+title: Updating Rails Data
+date: 2020-03-03
+layout: post
+image: https://placekitten.com/1920/1920
+custom_excerpt: This is intended to be a guide to updating Rails data using an inline table editor made with Vue
```
Comment (ctrlaltdelete44) [inline collections/_blog-posts/04-rails-with-vue-updating-rails-data.md]: No loner *intended* - the third installment in a series of...

File: collections/_blog-posts/04-rails-with-vue-updating-rails-data.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+  $ bin/webpack-dev-server
+    ...
+    > ℹ ｢wdm｣: Compiled successfully.
+  ```
+
+To enjoy your moment of glory, head to [http://localhost:3000/films](http://localhost:3000/films) and see your new table!
+
+## 2. Populating the table with Rails data
+
+The particularly eagle-eyed amongst you will notice that this article promises the use of Rails data - indeed, by the end of the [last article](blog-posts/03-rails-with-vue-accessing-rails-data.html) we already had access to our data. And now we're back to static data? Prepostorous!
```
Comment (ctrlaltdelete44) [inline collections/_blog-posts/04-rails-with-vue-updating-rails-data.md]: What

Comment (ctrlaltdelete44) [reply collections/_blog-posts/04-rails-with-vue-updating-rails-data.md]: Change this. This is weird

File: collections/_blog-posts/04-rails-with-vue-updating-rails-data.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+  <template v-slot:cell(title)='row'>
+    <b-form-input v-model='row.item.title' v-on:change.native="dataChanged"/>
+  </template>
+  ```
+
+  The moment of truth. Make some changes to your data, and check the console. If there are any errors then you're welcome to send me a very angry email. If not, refresh your page and see the change in data persisted. You can check this in the console, in other Vue components, wherever - you've just built an inline editor for Rails data!
+
+## 4. Conclusion
+
+The next steps in this are:
```
Comment (nodonnell98) [inline collections/_blog-posts/04-rails-with-vue-updating-rails-data.md]: next steps in this series will be

**Pull request merged**